### ISSUE TITLE
Drop support of Ruby 1.8 and 1.9

### ISSUE THREAD

**Issue by p0deje** [watir/watir-webdriver#324]
Both are dead. My suggestion is to update the code for 2.0 standards and update Travis configuration to run the specs on Ruby >= 2.0.

@watir/watir @watir/watir-webdriver-team What do you think?

**Comment by zeljkofilipin** [comment]
:+1:

**Comment by titusfortner** [comment]
Makes sense to me.
What code would need to be updated?

**Comment by p0deje** [comment]
Mostly style changes (e.g. hashrockets/lambda).

I also suggest to change old watir location args (two args) in favor of hashes in watirspec:

```ruby
browser.div(:id, 'one') # old
browser.div(id: 'one')  # new
```

**Comment by titusfortner** [comment]
Then I'm completely in favor of this. As someone who started using Ruby with 1.9 the (:id, 'one') is hard to look at. :)

**Comment by zeljkofilipin** [comment]
RuboCop can convert hashes from old syntax to the new one automatically.

**Comment by p0deje** [comment]
Yeah, I was planning to use Rubicon for that.

**Comment by zeljkofilipin** [comment]
@p0deje rubicon?

**Comment by p0deje** [comment]
@zeljkofilipin Sorry, didn't get it

**Comment by p0deje** [comment]
@zeljkofilipin Oh, that was some weird autocorrect. It's "Rubocop" :)

**Comment by p0deje** [comment]
Work has started in `ruby_2` branch. I'm planning to migrate watir-webdriver first, after that switch to watirpsec.

**Comment by titusfortner** [comment]
Fixed by #327
